benchbuild/utils/actions.py

```diff
@@ -17,23 +17,23 @@
 import functools as ft
 import itertools
 import logging
-import multiprocessing as mp
 import os
 import sys
 import textwrap
 import traceback
 from datetime import datetime
 
 import attr
+import pathos.multiprocessing as mp
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pathos.multiprocessing'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)